### PR TITLE
add logic for login

### PR DESCRIPTION
**Pull request by Vadym342**

### CODE REVIEW

File: client/src/pages/AuthorizationPage/components/Login/Login.jsx [hunk 1]
```diff
@@ -3,8 +3,31 @@ import Input from '@mui/material/Input';
...
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleFetchLoginUser = (fields) => {
+    dispatch(fetchLoginUser(fields));
+  };
+  
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    //!VALIDATION before request
```
Comment (OyakiMorse) [inline client/src/pages/AuthorizationPage/components/Login/Login.jsx]: unnecessary comment

File: client/src/pages/AuthorizationPage/components/Login/Login.jsx [hunk 2]
```diff
@@ -3,8 +3,31 @@ import Input from '@mui/material/Input';
...
+    dispatch(fetchLoginUser(fields));
+  };
+  
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    //!VALIDATION before request 
+
+    handleFetchLoginUser({
+      email: email,
+      password: password
```
Comment (OyakiMorse) [inline client/src/pages/AuthorizationPage/components/Login/Login.jsx]: if name of key and value is equal u can write as one property

**Review by OyakiMorse — APPROVED** [review]

**Review by OldFox94ua — APPROVED** [review]

File: client/src/pages/AuthorizationPage/components/Login/Login.jsx [hunk 3]
```diff
@@ -13,12 +36,31 @@ const Login = () => {
...
-            <Input name='email' type='email' pattern='[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2, 4}$' placeholder='Email'
-              sx={style.inputEmail}/>
-            <Input name='password' type='password' placeholder='Password' sx={style.inputPassword}/>
+          <form className={styles.authorizationForm} onSubmit={e => handleSubmit(e)}>
+            <Input 
+              name='email' 
+              type='email'
+              value={email}
+              onChange={e => setEmail(e.target.value)}
+              pattern='[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2, 4}$'
```
Comment (MattMsh) [inline client/src/pages/AuthorizationPage/components/Login/Login.jsx]: Move regular expressions into variables.

`const emailPattern = '[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2, 4}$';`

**Review by MattMsh — CHANGES_REQUESTED** [review]
### Rename file
It would be more appropriate to rename the **userRequest** file to **loginRequest**.
### Fix regexs

**Review by MattMsh — APPROVED** [review]

**Pull request merged**